Formula/libjwt.rb

```diff
@@ -0,0 +1,37 @@
+class Libjwt < Formula
+  desc "JSON Web Token C library"
+  homepage "https://github.com/benmcollins/libjwt"
+  url "https://github.com/benmcollins/libjwt/archive/v1.8.0.tar.gz"
+  sha256 "e97c15c14dbd49d5ba435cb26e06e6cdd6e32cf9f0be6b6e9a2d2d330923ea17"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "check" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jansson"
+  depends_on "openssl"
+
+  def install
+    system "autoreconf", "-v", "-i"
+    system "./configure", "--prefix=#{prefix}"
+    system "make", "all"
+    system "make", "check"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <stdlib.h>
+      #include <jwt.h>
+
+      int main() {
+        jwt_t *jwt = NULL;
+        if (jwt_new(&jwt) != 0) abort();
```

Comment:
probably best to avoid abort

Reply:
I understand that `abort()` is the way to make my test fail if the library is not working properly. Took the example from the `Formula/gmp.rb`, as suggested by @fxcoudert

Reply:
I think you can just return 1. The abort will trigger a log file to be written, I believe.

Reply:
👍